public/app/features/alerting/unified/components/receivers/EditReceiverView.tsx

```diff
@@ -18,9 +18,9 @@ export const EditReceiverView: FC<Props> = ({ config, receiverName, alertManager
   const receiver = config.alertmanager_config.receivers?.find(({ name }) => name === receiverName);
   if (!receiver) {
     return (
-      <InfoBox severity="error" title="Receiver not found">
+      <Alert severity="error" title="Receiver not found">
         Sorry, this receiver does not seem to exit.
```

Comment:
Typo in existing code?
```suggestion
        Sorry, this receiver does not seem to exist.
```

Reply:
Yep, it looks like this was never caught before 🙈